### PR TITLE
Create SORA Oracle Proposal

### PR DESCRIPTION
**Pull request by WRRicht3r**

### CODE REVIEW

**Review by stefashkaa — APPROVED** [review]

**Comment by takemiyamakoto** [comment]
File needs to be .md

**Comment by WRRicht3r** [comment]
Have uploaded the correct format in another PR, proceeding to close this one.

File: open_rfps/SORA Oracle Proposal [hunk 1]
```diff
@@ -0,0 +1,175 @@
+**Proposal Due Date**
+February 2022
```
Comment (takemiyamakoto) [inline open_rfps/SORA Oracle Proposal:2]: Um, it's already March. Make this August

File: open_rfps/SORA Oracle Proposal [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+**Proposal Overview**
+
+Implementation of oracles to provide relevant feeds of relatively small events about the offline world for on-chain usage such as TBC pricing, tokens linked to off-chain factors.
```
Comment (takemiyamakoto) [inline open_rfps/SORA Oracle Proposal:5]: Delete "small" in "small events"

**Pull request closed without merging**